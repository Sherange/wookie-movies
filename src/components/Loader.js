import React from 'react';
import {View, ActivityIndicator} from 'react-native';
import { primaryColor } from '../constants/theme';

const Loader = () => {
  return (
    <View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
      <ActivityIndicator size="large"  color={primaryColor}/>
    </View>
  );
};

export default Loader;
