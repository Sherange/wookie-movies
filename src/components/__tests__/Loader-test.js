import 'react-native';
import React from 'react';
import Loader from '../Loader';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  renderer.create(<Loader />);
});

test('Loader snapshot', () => {
  const snapshot = renderer.create(<Loader />).toJSON();
  expect(snapshot).toMatchSnapshot();
});
