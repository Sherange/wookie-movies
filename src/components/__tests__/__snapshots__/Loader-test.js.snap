// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    color="#D9251D"
    size="large"
  />
</View>
`;
